src/platform/react-hooks/src/hooks/useChannelInstance.ts

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { getContext } from '../AblyProvider.js';
+
+export function useChannelInstance(id: string, channelName: string) {
+  const channel = React.useContext(getContext(id))._channelNameToInstance[channelName];
+
+  if (!channel) {
+    throw new Error(
+      `Could not find a parent ChannelProvider in the component tree for name="${channelName}". Make sure your channel based hooks (usePresence, useChannel, useChannelStateListener) are called inside a <ChannelProvider> component`
```

Comment:
```suggestion
      `Could not find a parent ChannelProvider in the component tree for channelName="${channelName}". Make sure your channel based hooks (usePresence, useChannel, useChannelStateListener) are called inside a <ChannelProvider> component`
```

Reply:
This change got lost after [last rebase](https://github.com/ably/ably-js/compare/15b3be49e1e170dab4eb803eed85032cd278d075..5324354c8f9b3faf071055973c7f255ce7d7ac62#diff-1156ffeee8b809af8181f1f54cd4833e97e069209d361719f04f3df120f5b20dL9) on top of integration/v2 branch.
Need to reapply it.